polylogue/config.py

```
@@ -107,6 +107,7 @@ def persist_config(
     html_previews: bool,
     html_theme: str,
     index: Optional[IndexConfig] = None,
+    roots: Optional[Dict[str, Path]] = None,
```

Ruff (0.15.9)

[error] Undefined name `Dict`

(F821)